### DISCUSSION TITLE
Potential typo on video 16 - Modular Algebra - At the end of notebook Prime_Numbers

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#312] [Course: Foundations of Private Computation]
"""
For random 𝑎 the probability for it not being a witness is 25%, therefore the probability of not finding 10 witnesses at random if 𝑎 is composite is 0.25^10, i.e. If no witnesses are found in 10 rounds, then the probability for 𝑎 being prime is 1−0.2510= 0.999999
"""

I think "a" should be "**n**" in the two last mentions of "a" above

"""
For random 𝑎 the probability for it not being a witness is 25%, therefore the probability of not finding 10 witnesses at random if **n** is composite is 0.25^10, i.e. If no witnesses are found in 10 rounds, then the probability for **n** being prime is 1−0.2510= 0.999999
"""

**Comment by hwrdtm** [comment]
@gonzalo-munillag yes you are correct, we are trying to test whether **n** is a composite number or a prime number. Good catch!

**Reply by hwrdtm** [reply]
Captured in [this ticket](https://github.com/OpenMined/courses/issues/327), please feel free to mark this discussion as Answered!
